src/main/java/no/nav/tag/kontakt/oss/metrics/Metrics.java

```diff
@@ -1,38 +1,32 @@
 package no.nav.tag.kontakt.oss.metrics;
 
-import io.micrometer.core.instrument.Counter;
+import io.micrometer.core.instrument.MeterRegistry;
+import no.nav.tag.kontakt.oss.Kontaktskjema;
+import org.springframework.stereotype.Component;
 
+@Component
 public class Metrics {
-    private final Counter mottattKontaktskjemaSuccess;
-    private final Counter mottattKontaktskjemaFail;
-    private final Counter sendtGsakOppgaveSuccess;
-    private final Counter sendtGsakOppgaveFail;
+    private final MeterRegistry meterRegistry;
 
-    public Metrics(
-            Counter mottattKontaktskjemaSuccess,
-            Counter mottattKontaktskjemaFail,
-            Counter sendtGsakOppgaveSuccess,
-            Counter sendtGsakOppgaveFail
-    ) {
-        this.mottattKontaktskjemaSuccess = mottattKontaktskjemaSuccess;
-        this.mottattKontaktskjemaFail = mottattKontaktskjemaFail;
-        this.sendtGsakOppgaveSuccess = sendtGsakOppgaveSuccess;
-        this.sendtGsakOppgaveFail = sendtGsakOppgaveFail;
+    public Metrics(MeterRegistry meterRegistry) {
+        this.meterRegistry = meterRegistry;
     }
 
-    public void mottattKontaktskjema(boolean success) {
-        if (success) {
-            this.mottattKontaktskjemaSuccess.increment();
-        } else {
-            this.mottattKontaktskjemaFail.increment();
-        }
+    public void mottattKontaktskjema(boolean success, Kontaktskjema kontaktskjema) {
+        String counterName = success ? "mottatt_kontaktskjema_success" : "mottatt_kontaktskjema_fail";
+        meterRegistry.counter(
+                counterName,
+                "fylke", kontaktskjema.getFylke(),
+                "kommune", kontaktskjema.getKommunenr(),
+                "tema", kontaktskjema.getTema()
+        ).increment();
     }
 
     public void sendtGsakOppgave(boolean success) {
         if (success) {
-            this.sendtGsakOppgaveSuccess.increment();
+            meterRegistry.counter("sendt_gsakoppgave_success").increment();
```

Comment:
Hvorfor har vi byttet til strings for å send metrikker, og ikke "typesette" dem med metoder?

Reply:
Det har alltid vært strings, bare trukket ut til en config-klasse. Grunnen til at det ble gjort var en misforståelse av .counter-funksjonen. Kan forklare med ved behov